src/SeleniumLibrary/keywords/webdrivertools/webdrivertools.py

```diff
@@ -228,7 +228,7 @@ def _geckodriver_log(self):
         log_file = self._get_log_path(
             os.path.join(self.log_dir, "geckodriver-{index}.log")
         )
-        logger.info(f"Firefox driver log is always forced to to: {log_file}")
+        logger.trace(f"Firefox driver log is always forced to to: {log_file}")
```

Comment:
Should this be debug as compared to trace level? I'm pretty sure we don't have published guidelines so almost asking myself here why one over the other ..

Reply:
Having this at info or debug breaks a whole bunch of acceptance tests, I didn't see an obvious way to make robotstatuschecker ignore these.

Reply:
Makes sense and good enough reason (for now) to drop these down into trace level